python/pysrc/kaskada/__init__.py

```diff
@@ -21,5 +22,6 @@
     "Result",
     "sources",
     "Timestream",
+    "udf",
     "windows",
 ]
```

Comment:
I'm not quite 100% on what _needs_ to be here or not

Reply:
This is the list of "public" things that are available as `kd.<name>` when you `import kaskada as kd`. So, the `udf` annotatino probably needs to be here at this point. Not sure the `Udf` needs to, if the user isn't implementing the class directly yet.